styles.css

```diff
@@ -1755,6 +1847,204 @@
+    top: 0%;
+    padding-top: 7%;
+    display: flex;
+    justify-content: center;
+    overflow-y: scroll !important;
+  }
+
+  .card-pops {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 12px;
+    width: 85%;
+    height: 964px;
+    background: #ffff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+  }
+
+  .containar {
+    width: 100%;
+    height: 100%;
+  }
+
+  .title {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    height: auto;
+  }
+
+  .card-title-pops {
+    display: flex;
+    margin-top: 1%;
+    margin-left: 2%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    height: auto;
+  }
+
+  .fa-times {
+    color: #67798e;
+    margin-top: 3%;
+    margin-right: 4%;
+  }
+
+  #pops {
+    margin-top: -3%;
+    margin-left: 2%;
+  }
+
+  .card-list-pops {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 10px;
+    width: 236px;
+    height: auto;
+    margin-left: -73%;
+  }
+
+  .card-img-pops {
+    width: 96%;
+    height: auto;
+    padding: 1%;
+    margin-top: 0%;
+  }
+
+  .project-desktop {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 0;
+    margin-top: 5%;
+    width: 45%;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    background-color: #fff;
+    cursor: pointer;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+  }
+
+  .project-desktop:hover {
+    background-color: #7f8cff;
+    color: white;
+  }
+
+  .project-desktop:active {
+    background: #2230d2;
+    color: white;
+  }
+
+  .tags-pops {
+    display: flex;
+    flex-direction: row;
+    padding: 0;
+    gap: 4%;
+    width: 90%;
+    height: 24px;
+  }
+
+  #lblock {
+    width: 100%;
+    padding-bottom: 14%;
+    margin-top: 2%;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  #ptpop {
+    margin: 0;
+    flex-direction: row;
+    width: 100%;
+  }
+
+  #lorem {
+    width: 62%;
+    height: auto;
+    margin-right: 3%;
+  }
+
+  .rightb {
+    width: 30%;
+  }
+
+  #java {
+    width: auto;
+  }
+
+  #color {
+    color: #396df2;
+  }
+
+  .pop-lang {
+    display: flex;
+    flex-direction: row;
+    gap: 3%;
+    padding: 0%;
+    margin-left: 0%;
+  }
+
+  #rubi {
+    margin-left: 0%;
+    width: auto;
+    padding: 1px 12px;
+  }
+
+  #source2 {
+    margin-top: 3%;
+    font-size: 13px;
+    padding: 2px 5px;
+    color: #396df2;
+  }
+
+  #live2 {
+    margin-top: 3%;
+    font-size: 14px;
+    color: #396df2;
+    padding: 2px 5px;
+  }
+
+  .buttons {
```

Comment:
- [ ] Based on your styling. It's pretty good. But one suggestion. Please add some space at the bottom of your description. Kindly see the picture:

<img width="1261" alt="Screenshot 2023-03-18 at 08 42 33" src="https://user-images.githubusercontent.com/69011963/226083306-835baad9-1d08-43c4-a6d3-d41605291f71.png">